tutorials/10-bayesian-differential-equations/10_bayesian-differential-equations.jmd

```diff
@@ -78,6 +78,19 @@ plot(sol; alpha=0.3)
 scatter!(sol.t, odedata'; color=[1 2], label="")
 ```
 
+# alternatively, we can use real-world data from Hudson’s Bay Company records (a Stan implementation can be found here: https://mc-stan.org/users/documentation/case-studies/lotka-volterra-predator-prey.html)
+
+# years = [1900,1901,1902,1903,1904,1905,1906,1907,1908,1909,1910,1911,1912,1913,1914,1915,1916,1917,1918,1919,1920]
+
+# prey
+
+hare = [30.0, 47.2, 70.2, 77.4, 36.3, 20.6, 18.1, 21.4, 22.0, 25.4, 27.1, 40.3, 57.0, 76.6, 52.3, 19.5, 11.2, 7.6, 14.6, 16.2, 24.7]
+
+# predator
+
+lynx = [4.0, 6.1, 9.8, 35.2, 59.4, 41.7, 19.0, 13.0, 8.3, 9.1, 7.4, 8.0, 12.3, 19.5, 45.7, 51.1, 29.7, 15.8, 9.7, 10.1, 8.6]
+odedata = [hare lynx]'
```

Comment:
This should be put inside of a code block, shouldn't it? Or we should only add the comment?

Reply:
> This should be put inside of a code block, shouldn't it? Or we should only add the comment?

@devmotion Good point, yes it should sit in the `julia` block. Just fixed it :)